ipc/provider/src/lib.rs

```diff
@@ -722,11 +724,10 @@ impl IpcProvider {
         conn.manager().list_bootstrap_nodes(subnet).await
     }
 
-    /// Returns the latest finality from the parent committed in a child subnet.
-    pub async fn latest_parent_finality(&self, subnet: &SubnetID) -> anyhow::Result<ChainEpoch> {
+    /// Returns the latest parent state from the parent committed in a child subnet.
+    pub async fn latest_parent_state(&self, subnet: &SubnetID) -> anyhow::Result<ChainEpoch> {
```

Comment:
Could you elaborate what this is? Is it a gossiped block that was imported but not finalized? How do we handle competing forks or is that not a concern at this level?